spec/invoker.e2e.spec.js

```diff
@@ -29,9 +33,9 @@ describe('invoker =>', () => {
 
         describe(`with a ${testCase.interactionType} square function =>`, () => {
 
-            beforeEach(() => {
+            beforeEach(async () => {
                 jasmine.addCustomEqualityTester(outputSignalCustomEqual);
-                ({server, address} = tryStartInvoker(testCase.functionUri));
+                ({invocation, address} = await tryStartInvoker(globalScope, testCase.functionUri));
```

Comment:
remove the extra parens (many instances of this)
```suggestion
                {invocation, address} = await tryStartInvoker(globalScope, testCase.functionUri);
```

Reply:
eslint complains if they are not here:
```
~/node-function-invoker/spec/invoker.e2e.spec.js
  38:39  error  Parsing error: Unexpected token =
```